cluster/saltbase/salt/etcd/init.sls

```diff
@@ -34,6 +26,28 @@ delete_etcd_initd:
   file.absent:
     - name: /etc/init.d/etcd
 
+#stop legacy etcd_service
+stop_etcd-service:
+  service.dead:
+    - name: etcd
+    - enable: None
+
+{% endif %}
+
+touch /var/log/etcd.log:
+  cmd.run:
+    - creates: /var/log/etcd.log
+
+/var/etcd:
```

Comment:
Should this be in an else block? For the platforms that don't have a host etcd, we don't need this rule to execute.